p/protobuf/protobuf_342cbe0a0415_ubi8.4.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+# -----------------------------------------------------------------------------
+#
+# Package       : gogo/protobuf
+# Version       : 342cbe0a0415
+# Source repo   : https://github.com/gogo/protobuf
+# Tested on     : RHEL ubi 8.4
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Apurva Agrawal<Apurva.Agrawal3@ibm.com>
+#
+# Disclaimer    : This script has been tested in root mode on given
+# ==========      platform using the mentioned version of the package.
+#                 It may not work as expected with newer versions of the
+#                 package and/or distribution. In such case, please
+#             	  contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+PACKAGE_NAME=github.com/gogo/protobuf
+PACKAGE_VERSION=342cbe0a0415
```

Comment:
Can you make this PACKAGE_VERSION as configurable ?

something like :

`PACKAGE_VERSION=${1:-342cbe0a0415}`

Reply:
Resolved review comments.